backend/src/main/java/dev/tripdraw/dto/trip/TripsReadResponse.java

```diff
@@ -0,0 +1,21 @@
+package dev.tripdraw.dto.trip;
+
+import dev.tripdraw.domain.trip.Trip;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.List;
+
+public record TripsReadResponse(
+        @Schema(description = "여행 목록")
+        List<TripReadResponse> trips
+) {
+
+    public static TripsReadResponse from(List<Trip> trips) {
+        return new TripsReadResponse(generateTripReadResponses(trips));
```

Comment:
collectingAndThen을 사용하는 방법도 있을 것 같아요 👍

Reply:
```java
private static List<TripReadResponse> generateTripReadResponses(List<Trip> trips) {
        return trips.stream()
                .map(TripReadResponse::from)
                .collect(Collectors.collectingAndThen(Collectors.toList(), List::copyOf));
    }
```
제가 이해한 바가 맞다면 이걸 쓰라는 말이겠죠? 아니라면 다시 커멘트 부탁드려요

Reply:
맞아요 사용시 Collectors는 static import하면 더 좋을 것 같아용~

Reply:
```suggestion
        return trips.stream()
                .map(TripReadResponse::from)
                .collect(Collectors.collectingAndThen(Collectors.toList(), TripsReadResponse::new));
```

이거 될지 모르겠지만 확인 부탁드려융

Reply:
@greeng00se 와 맛있따 리팩터링 했습니다